src/components/data_comps/datm/cime_config/namelist_definition_datm.xml

```diff
@@ -1616,8 +1628,6 @@
       scientific modes supported by the model, they are listed below.  The
       mode is selected by a character string set in the strdata namelist
       variable dataMode.
-      datamode = "CPLHIST"
```

Comment:
I think CPLHIST should also be removed from the list of valid values for this variable; do you agree?

Reply:
Yes! Good catch.